ts/twitter/src/app/home/_components/timeline/use-timeline-feed.ts

```diff
@@ -7,9 +7,18 @@ interface useTimelineFeedReturn {
   posts: FollowingPostsResponse;
 }
 
+export enum EventType {
+  TimelineAccessed = "TimelineAccessed",
+  PostCreated = "PostCreated",
+  PostDeleted = "PostDeleted",
+}
+
+export interface EventResponse {
```

Comment:
I think the `EventResponse` is a bit ambiguous.
```suggestion
export interface TimelineEvent {
```